resources/assets/lib/components/beatmapset-event.tsx

```diff
@@ -43,7 +43,8 @@ export default class BeatmapsetEvent extends React.PureComponent<Props> {
   }
 
   private get firstPost() {
-    return this.discussion?.starting_post ?? this.discussion?.posts?.[0];
+    const post = this.discussion?.starting_post ?? this.discussion?.posts?.[0];
+    return post?.system ? null : post;
```

Comment:
I wonder if `startingPost` should just always return `BeatmapsetDiscussionMessagePostJson` (but nullable?)